### PR TITLE
Vitamin a and zinc multiplication models

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

File: multiplication_models/functions_for_all_nutrients.py
Comment (beatrixkh) [inline multiplication_models/functions_for_all_nutrients.py:26]: Oh oops I didn't get back to you on this @alibow ! I took this function and broke it up into a few sub-functions that do each step of this, and allow me to pull out some interim tables before it generates the full location-year-coverage_level table.

I could wait until this is merged, and then PR off of this with those? or open up a new PR right now so you can see, and edit it once this branch is merged

File: multiplication_models/functions_for_all_nutrients.py
Comment (aflaxman) [inline multiplication_models/functions_for_all_nutrients.py:51]: This logic would be clearer to me if you grouped it differently.  Perhaps @beatrixkh has already done this, so remind me if it doesn't change after her PR and we can talk through alternative ways to do it.

File: multiplication_models/functions_for_all_nutrients.py
Comment (aflaxman) [inline multiplication_models/functions_for_all_nutrients.py]: Let's use (upper - lower) / (2 * 1.96) instead, even though it is also not perfect.

File: multiplication_models/functions_for_all_nutrients.py
Comment (aflaxman) [inline multiplication_models/functions_for_all_nutrients.py:76]: I have a theory that it is cooler to make a list of DataFrames and then pd.concat the whole list, instead of repeatedly concating in a loop.  Writing this comment has made me realize that I have not tested my theory, though.

File: multiplication_models/functions_for_all_nutrients.py
Comment (aflaxman) [inline multiplication_models/functions_for_all_nutrients.py]: It is clearer to do all the imports at the beginning of the .py file

**Review by aflaxman — APPROVED** [review]
I only made it through your first .py file in detail, but I thought I'd pass on my feedback rather than wait to get through the rest carefully.

File: multiplication_models/functions_for_all_nutrients.py
Comment (alibow) [reply multiplication_models/functions_for_all_nutrients.py:26]: Ah, I see! I think we can both just merge our branches without breaking anything and then afterwards I will update them all to your easier to follow updated version if that sounds good? @beatrixkh

**Pull request merged**
